applications/monitoring/values.yaml

```diff
@@ -24,50 +122,64 @@ chronograf:
       memory: 30Gi
       cpu: 4
 
-  ## Use our own Gafaelfawr ingress.
+  # -- Chronograf ingress
   ingress:
-    ## We will use Gafaelfawr ingresses instead
     enabled: false
+    tls: false
+    hostname: ""
+    className: "nginx"
+    path: /chronograf(/|$)
 
   ## Enable OAuth
+  # -- Enable Chronograf oauth
+  # Never enable it: it breaks the deployment because it's expecting a static
+  # TOKEN_SECRET.  Instead, to get oauth working, leave this setting as-is
+  # and just configure all the correct environment variables (see below).
```

Comment:
There's only one setting, so the documentation should move down to be a comment on enabled.